tests/testsuite/locate_project.rs

```diff
@@ -57,22 +67,33 @@ fn workspace() {
         .file("inner/src/lib.rs", "")
         .build();
 
-    let outer_manifest = r#"{"root": "[ROOT]/foo/Cargo.toml"}"#;
-    let inner_manifest = r#"{"root": "[ROOT]/foo/inner/Cargo.toml"}"#;
```

Comment:
Guess it's also possible to do something alike? By extracting them as variables, we can make sure invocations that are ought to be the same still get the same JSON.

```suggestion
    let outer_manifest = str![[r#"{"root": "[ROOT]/foo/Cargo.toml"}"#]];
    let inner_manifest = str!r#"{"root": "[ROOT]/foo/inner/Cargo.toml"}"#]];
```

Reply:
It appears the suggested change might have a typo
```rust
let inner_manifest = str![[r#"{"root": "[ROOT]/foo/inner/Cargo.toml"}"#]];
```